examples/show-progress/go.sql

```diff
@@ -0,0 +1,13 @@
+-- can disable the spinner to show only progress bar
+SELECT 'loader-start' AS component,  'non-existent-class' AS spinner;
```

Comment:
This is a bit janky. I'm thinking instead of default spinner, user must specify a tabler spinner. If NULL then no spinner.